packages/atomic/src/pages/examples/folding.html

```diff
@@ -76,6 +86,365 @@
           <atomic-layout-section section="results">
             <atomic-smart-snippet></atomic-smart-snippet>
             <atomic-folded-result-list image-size="small">
+              <atomic-result-template must-match-sourcetype="Slack">
+                <template>
+                  <style>
+                    div.result-root.with-sections.display-list.density-normal atomic-result-section-visual {
+                      margin-right: 16px;
+                    }
+
+                    .slack-section-badge {
+                      align-items: center;
+                      display: flex;
+                      width: 100%;
+                    }
+
+                    .slack-title-icon {
+                      display: flex;
+                      width: 100%;
+                      margin-top: 8px;
+                    }
+
+                    .slack-badge::part(result-badge-element) {
+                      background-color: #86d8ff;
+                      color: white;
+                      border-radius: 32px;
+                      padding-top: 2px;
+                      padding-bottom: 2px;
+                    }
+
+                    ::part(children-root) {
+                      margin-left: 70px;
+                      border-left: thin solid #bcc3ca;
```

Comment:
it's subjective, but we generally use `var(--atomic-neutral)` for borders, so I think it would be a good idea to use it here too.

Reply:
Didn't know about the `var(--...`